src/app/messages-page/create-access-group/create-access-group.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="modal-header">
+  <div class="modal-header-x" (click)="bsModalRef.hide()">
+    <i-feather name="x"></i-feather>
+  </div>
+</div>
+<div class="likes-modal-container">
+  <header class="border-bottom p-2">
+    <h1 class="fs-17px font-weight-bold">Create Group Chat</h1>
+  </header>
+  <form class="p-2" (submit)="this.createGroupChat($event)">
+    <div class="form-group">
+      <label for="accessGroupName">Group Name</label>
+      <input
+        type="text"
+        id="accessGroupName"
+        name="accessGroupName"
+        class="form-control"
+        [(ngModel)]="this.groupName"
+      />
+    </div>
+    <div class="form-group" [ngStyle]="{ height: '72px' }">
+      <label for="accessGroupMembers">Add Group Members</label>
+      <search-bar
+        [showCloutavista]="false"
+        [isSearchForUsersToMessage]="true"
+        [startingSearchText]=""
+        (creatorToMessage)="this.onSearchItemSelected($event)"
```

Comment:
should I remove this. ?

Reply:
Let's remove just for consistency's sake